FitNesseRoot/DbFit/AcceptanceTests/JavaTests/Db2Tests/FlowMode/properties.xml

```diff
@@ -11,6 +11,7 @@
 <Suite/>
 <Suites>(Shared Tests)</Suites>
 <SymbolicLinks>
+<CommitAndRollback>.DbFit.AcceptanceTests.LibraryTests.CommitAndRollback</CommitAndRollback>
```

Comment:
There's probably a better name out there somewhere. I was lacking inspiration at the time. Any offers accepted!

The idea was to create a library of tests that can be reused/included where appropriate (where the database supports the feature). Half way between the common `CoreTests` and the tests that have been duplicated, for different DB types, that we still have hanging around.